app/src/main/java/org/wikipedia/analytics/ABTest.kt

```diff
@@ -19,6 +20,10 @@ open class ABTest(private val abTestName: String, private val abTestGroupCount:
 
     protected open fun assignGroup() {
         testGroup = Random(System.currentTimeMillis()).nextInt(Int.MAX_VALUE).mod(abTestGroupCount)
+        ActivityTabEvent.submit(activeInterface = "NULL/NOT NEEDED", action = "group_assign", group = when (testGroup) {
```

Comment:
I checked the deck, and it looks like you can leave the `activeInterface = null` instead of capital letters. You may check with Shay about this.